chrome/browser/ui/views/frame/browser_frame_mus.cc

```diff
@@ -85,12 +85,20 @@ bool BrowserFrameMus::UsesNativeSystemMenu() const {
 }
 
 bool BrowserFrameMus::ShouldSaveWindowPlacement() const {
+#if !defined(USE_OZONE) || !defined(OS_LINUX) || defined(CHROMEOS)
```

Comment:
maybe an easier ifdef is the otherway around:

#if defined(USE_OZONE) && defined(OS_LINUX) ?
  ...
#else 
  return false;
#endif